app/src/main/java/ar/nablab/mimerendero/repositories/MapRepositoryImpl.kt

```diff
@@ -0,0 +1,71 @@
+package ar.nablab.mimerendero.repositories
```

Comment:
maybe we could have a mock data build variant with a "mock" MapRepositoryImpl so there is no need to load fake data we could just use a simple in memory map with fake places.